ta_services/keymaster/include/keymaster_ta.h

```diff
@@ -7,6 +7,8 @@
 #define TA_KEYMASTER_UUID { 0xd36b30c7, 0x5a5f, 0x472c, \
 			    {0xaf, 0x97, 0x7f, 0x38, 0xa2, 0xed, 0xab, 0x7d } }
 
+#define KEYMASTER_CMD_CONFIGURE		0
```

Comment:
Please add a description of this TAF. I think I'd prefer a more specific name, for example `KEYMASTER_CMD_SET_OS_VERSION`.